docs/data/sdks/android-kotlin/index.md

```diff
@@ -43,16 +52,15 @@ The SDK internally uses a few Java 8 language APIs through desugaring. Make sure
 
 ### Step 3: Initialization
 
-Before you can instrument, you must initialize the SDK using the API key for your Amplitude project. Amplitude recommends doing the initialization in the Main Activity, which never gets destroyed, or the Application class if you have one. After it's initialized, you can use the Android SDK anywhere in your Android application.
+Before you can instrument, you must initialize the SDK using the [API key](https://www.docs.developers.amplitude.com/analytics/find-api-credentials/) for your Amplitude project. Amplitude recommends doing the initialization in the Main Activity, which never gets destroyed, or the Application class if you have one. After it's initialized, you can use the Android SDK anywhere in your Android application.
```

Comment:
```suggestion
Before you can instrument, you must initialize the SDK using the [API key](../analytics/find-api-credentials/) for your Amplitude project. Amplitude recommends doing the initialization in the Main Activity, which never gets destroyed, or the Application class if you have one. After it's initialized, you can use the Android SDK anywhere in your Android application.
```

We use relative links to other developer docs. They can be really tricky in this project so if you have any issues with them, slack me.

Reply:
Resolved as changed to relative links in quickstart guide